src/books/dto/update-book.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { CanBeUndefined } from '../../utilities/can-be-undefined';
+
+export class UpdateBookDto {
+  @IsString()
+  @IsNotEmpty()
+  title?: string;
+
+  @CanBeUndefined()
+  @IsNumber({}, { each: true })
+  authorsIds?: number[];
```

Comment:
In `CreateBookDto` you are using `authorIds` but here you are using `authorsIds`. Be consistent.